app/src/main/kotlin/edu/stanford/bdh/engagehf/navigation/screens/AppScreenViewModel.kt

```diff
@@ -84,8 +120,15 @@ class AppScreenViewModel @Inject constructor(
 data class AppUiState(
     val items: List<BottomBarItem>,
     val selectedItem: BottomBarItem,
-    val isBottomSheetExpanded: Boolean = false,
     val bottomSheetContent: BottomSheetContent? = null,
+    val accountUiState: AccountUiState = AccountUiState(),
+)
+
+data class AccountUiState(
+    val showDialog: Boolean = false,
+    val userInfo: UserInfo = UserInfo("", null),
```

Comment:
Thanks for applying this 💪 

`UserInfo` was meant to be a domain object, we can decorate it with other user properties which shouldn't all be necessary for the the account ui state. Here we are on a ui layer and we should only store the properties that we need to display in the UI. TLDR: just replace this with two properties `email` and `displayName`. QQ: I see `initials` never get's set. Can we remove it?

Reply:
Good that you saw that. The test account had no names so I never realised it wasn't set and only saw it working in the previews!